src/test/java/com/proofpoint/discovery/store/TestInMemoryStore.java

```diff
@@ -179,6 +179,6 @@ public void testDoesntDefaultMaxAge()
 
     private static Entry entryOf(List<Service> value, long timestamp)
     {
-        return entry(NODE_ID.getBytes(), value, timestamp, 60_000L);
+        return entry(NODE_ID.getBytes(), value, timestamp, 60_000L, "127.0.0.1");
     }
 }
```

Comment:
Shouldn't there be a test of the added `get(Id<Node>)` method?

Reply:
...ah, that's only relevant to `TestInMemoryDynamicStore`, which inherits all tests of `TestDynamicStore`. So no issue.